src/vector/oscal-component.yaml

```diff
@@ -78,9 +78,9 @@ component-definition:
                   Event outcomes can include indicators of event success or failure and event-specific results (e.g., the security state of the information system after the event occurred).
 
                 # Control Implementation
-                  Logs are captured by promtail from the node. The node logs will contain the necessary log data from all pods/applications inside the selected nodes.
-                  Validating `logfmt` as the config.logFormat would be the goal. This is currently a secret mounted to /etc/promtail/promtail.yaml in the promtail container. We will ensure the promtail.yaml file is at a minimum the target config.
-                https://grafana.com/docs/loki/latest/send-data/promtail/stages/logfmt/
+                  Logs are captured by vector from the node. The node logs will contain the necessary log data from all pods/applications inside the selected nodes.
+                  Validating `logfmt` as the config.logFormat would be the goal. This is currently a secret mounted to /etc/vector/vector.yaml in the vector container. We will ensure the vector.yaml file is at a minimum the target config.
+                https://grafana.com/docs/loki/latest/send-data/vector/stages/logfmt/
```

Comment:
remove this link?

Reply:
Yeah absolutely - I did an initial find+replace of promtail -> vector, but expect that @CloudBeard might have more changes on this one. Didn't spend too much effort on the OSCAL in particular because of that.